Python/Product/PythonTools/PythonTools/OutliningTaggerProvider.cs

```diff
@@ -103,7 +103,7 @@ internal class OutliningTagger : ITagger<IOutliningRegionTag> {
 
                 var tags = await entry.Analyzer.GetOutliningTagsAsync(snapshot);
                 if (tags != null) {
-                    _tags = tags.Concat(ProcessRegionTags(snapshot)).ToArray();
+                    _tags = tags.Concat(ProcessRegionTags(_buffer.CurrentSnapshot)).ToArray();
```

Comment:
The issue here is that the buffer may have been modified during the await call.  So we capture the current snapshot which holds the earliest possible version we need to translate from.  But then we need to apply the changes to the current version, not the earliest possible version.